.github/workflows/docker-build-push-github.yaml

```diff
@@ -7,6 +7,15 @@ name: Package OCI
 on:
   # This entry allows calling this workflow from other workflows
   workflow_call:
+    inputs:
+      docker-build-context:
```

Comment:
This is redundant. What if we rename it to: `context`.

Reply:
Similar to my other comment, I find the `docker` prefix inoffensive. We can drop the `build` word from it.

Reply:
Let's use `docker-context` then!